{
 "bad": false,
 "new_version": "0.15.0",
 "new_version_attempts": {
  "0.10.0": 11,
  "0.11.0": 11,
  "0.12.0": 33,
  "0.13.0": 50,
  "0.14.0": 1402,
  "0.15.0": 114,
  "0.9.0": 45
 },
 "new_version_errors": {
  "0.10.0": "We found a problem parsing the recipe: \n\nwhile constructing a mapping\n  in \"<unicode string>\", line 5, column 3:\n      name: <{ name }}-suite\n      ^ (line: 5)\nfound duplicate key \"version\" with value \"<{ GIT_DESCRIBE_TAG }}\" (original value: \"<{ version }}\")\n  in \"<unicode string>\", line 10, column 3:\n      version: <{ GIT_DESCRIBE_TAG }}\n      ^ (line: 10)\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n",
  "0.11.0": "We found a problem parsing the recipe: \n\nwhile constructing a mapping\n  in \"<unicode string>\", line 5, column 3:\n      name: <{ name }}-suite\n      ^ (line: 5)\nfound duplicate key \"version\" with value \"<{ GIT_DESCRIBE_TAG }}\" (original value: \"<{ version }}\")\n  in \"<unicode string>\", line 10, column 3:\n      version: <{ GIT_DESCRIBE_TAG }}\n      ^ (line: 10)\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n",
  "0.12.0": "We found a problem parsing the recipe: \n\nwhile constructing a mapping\n  in \"<unicode string>\", line 5, column 3:\n      name: <{ name }}-suite\n      ^ (line: 5)\nfound duplicate key \"version\" with value \"<{ GIT_DESCRIBE_TAG }}\" (original value: \"<{ version }}\")\n  in \"<unicode string>\", line 10, column 3:\n      version: <{ GIT_DESCRIBE_TAG }}\n      ^ (line: 10)\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n",
  "0.13.0": "We found a problem parsing the recipe: \n\nwhile constructing a mapping\n  in \"<unicode string>\", line 5, column 3:\n      name: <{ name }}-suite\n      ^ (line: 5)\nfound duplicate key \"version\" with value \"<{ GIT_DESCRIBE_TAG }}\" (original value: \"<{ version }}\")\n  in \"<unicode string>\", line 10, column 3:\n      version: <{ GIT_DESCRIBE_TAG }}\n      ^ (line: 10)\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n",
  "0.14.0": "We found a problem parsing the recipe: \n\nwhile constructing a mapping\n  in \"<unicode string>\", line 5, column 3:\n      name: <{ name }}-suite\n      ^ (line: 5)\nfound duplicate key \"version\" with value \"<{ GIT_DESCRIBE_TAG }}\" (original value: \"<{ version }}\")\n  in \"<unicode string>\", line 10, column 3:\n      version: <{ GIT_DESCRIBE_TAG }}\n      ^ (line: 10)\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n",
  "0.15.0": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/10400316171\">bot CI job</a>): main:\nError running migrate-feedstock in container - error VersionMigrationError raised:\n'The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '0.15.0' to make sure they exist!\n\nWe also found the following errors:\n\n - We found a problem parsing the recipe for version '0.15.0': \n\nDuplicateKeyError('while constructing a mapping',   in \"<unicode string>\", line 5, column 3:\n      name: <{ name }}-suite\n      ^ (line: 5), 'found duplicate key \"version\" with value \"<{ GIT_DESCRIBE_TAG }}\" (original value: \"<{ version }}\")',   in \"<unicode string>\", line 10, column 3:\n      version: <{ GIT_DESCRIBE_TAG }}\n      ^ (line: 10), '\\n                    To suppress this check see:\\n                        http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\\n                    ', '                    Duplicate keys will become an error in future releases, and are errors\\n                    by default when using the new API.\\n                    ')\n\ntraceback:\n  File \"/opt/autotick-bot/conda_forge_tick/update_recipe/version.py\", line 415, in update_version\n    cmeta = CondaMetaYAML(raw_meta_yaml)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/autotick-bot/conda_forge_tick/recipe_parser/_parser.py\", line 525, in __init__\n    self.meta = self._parser.load(\"\".join(lines))\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/main.py\", line 451, in load\n    return constructor.get_single_data()\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 116, in get_single_data\n    return self.construct_document(node)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 125, in construct_document\n    for _dummy in generator:\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 1476, in construct_yaml_map\n    self.construct_mapping(node, data, deep=True)\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 1365, in construct_mapping\n    value = self.construct_object(value_node, deep=deep)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 147, in construct_object\n    data = self.construct_non_recursive_object(node)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 188, in construct_non_recursive_object\n    for _dummy in generator:\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 1476, in construct_yaml_map\n    self.construct_mapping(node, data, deep=True)\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 1366, in construct_mapping\n    if self.check_mapping_key(node, key_node, maptyp, key, value):\n       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/ruamel/yaml/constructor.py\", line 278, in check_mapping_key\n    raise DuplicateKeyError(*args)\n\n'",
  "0.9.0": "We found a problem parsing the recipe: \n\nwhile constructing a mapping\n  in \"<unicode string>\", line 5, column 3:\n      name: <{ name }}-suite\n      ^ (line: 5)\nfound duplicate key \"version\" with value \"<{ GIT_DESCRIBE_TAG }}\" (original value: \"<{ version }}\")\n  in \"<unicode string>\", line 10, column 3:\n      version: <{ GIT_DESCRIBE_TAG }}\n      ^ (line: 10)\n\nTo suppress this check see:\n    http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n\nDuplicate keys will become an error in future releases, and are errors\nby default when using the new API.\n"
 }
}